torch_xla/core/xla_model.py

```diff
@@ -409,6 +409,29 @@ def all_to_all(value,
   return result[0]
 
 
+def collective_permute(value, pairs):
+  """Performs a XLA `CollectivePermute()` operation on the input tensor.
+
+  See: https://www.tensorflow.org/xla/operation_semantics#collectivepermute
+
+  Args:
+    value (torch.Tensor): The input tensor.
+    pairs (list): A list of (source_replica_id, target_replica_id) pairs,
+      representing the sender and receiver for the `collective_permute()`
```

Comment:
Indenting looks a bit odd. Did you run `google-pyformat`?

Reply:
`google-pyformat` will behave differently if I manually added some line breaks before I run. Fix this one along with couple comments.